### ISSUE TITLE
Homepage

### ISSUE THREAD

**Issue by dalehamel** [bpftools/bpf.sh#9]
#8 Improves the homepage to make it come from markdown and a template.

We should further enhance the template to make the front page nicer, or change the generator for the front page to something like hugo, jekyll, or whatever we'd like.

This is issue is to try to decide what we want on the page, and how to build it / what builder and theme to use for it.

@fntlnz @leodido

**Comment by fntlnz** [comment]
I like the actual home page better than the previous one, however it would be cool if it was mobile optimized. Maybe we could start with the theme I made for fntlnz.wtf and personalize it for bpf.sh?

**Comment by dalehamel** [comment]
@fntlnz yeah should be easy enough, just need to get an html template outa it, it's a pandoc template format but all of those are pretty similar.
